shell/src/main/java/tachyon/shell/TfsShell.java

```diff
@@ -138,8 +138,8 @@ public int run(String... argv) {
     try {
       command.run(args);
       return 0;
-    } catch (IOException ioe) {
-      System.out.println(ioe.getMessage());
+    } catch (IOException e) {
+      System.out.println(e.getMessage());
```

Comment:
I wonder if we should do e.printStackTrace() here, possibly guarded by a command-line flag to shell (e.g. -debug).

When shell does not work, the message does not provide information as to the root cause of the problem. I have helped debug shell issues and the first thing I typically do here is that I replace this line with e.printStackTrace().